### ISSUE TITLE
C3D feature +linear SVM

### ISSUE THREAD

**Issue by erichouyi** [facebook/C3D#99]
hello!  I use the C3D to extract video descriptors of Weizmann(10 classes, 93videos) and UCF11(11 classes, 1597 videos)dataset, and take linear SVM as classifier. The recognition accurancy for Weizmann is about 87%(leave one out).However,  the accurancy for UCF11 is nearly 99% (4-fold cross validation),and I feel  unbelievable. Do you think it is true or normal ?

**Comment by firmamentqj** [comment]
Hi, I think this is normal. Because the pre-trained model is based on the Sports1M dataset, which is a large-scale realistic dataset. However, Weizmann includes simple actions under controlled environment. So without fine-tuning, the extract features may not be perfect for Weizmann.

Cheers,
Nigel

**Comment by dutran** [comment]
@erichouyi This seems weird to me. In my paper we reported C3D (fc6) + linear SVM around 82-83%. Also, for UCF101, the published split is 3-fold cross validation. I am not sure how you can use 4-fold. Be careful, because you may end up with mixing-up the train and test split.

**Comment by erichouyi** [comment]
@dutran Thank you for your suggestion. I have checked the train and test split, and they do not mix up. I just wonder why fc6 + linear SVM can reach so high accuracy in UCF11 (although it only has1597 videos), comparing to iDT features.

**Comment by erichouyi** [comment]
Hi, @dutran ! Recently, i do the fine-tuning example on UCF 101 and UCF 11. For UCF 101, the average clips accuracy is 80.22% (it is close to your result ). For UCF 11 (13,280 clips for train, 5,210 for test), the average clips accuracy is 95.31% , although this is the clip-level accuracy, it is higher than Dense Trajectories feature [1] (85.4%). is it normal or there is something wrong with my experiment ?

[1] Dense Trajectories and Motion Boundary Descriptors for Action Recognition, 2013, IJCV.

**Comment by KervinBill** [comment]
@dutran  ''UCF101, the published split is 3-fold cross validation. ''  how did you do the 3-fold cross validation?

**Comment by dutran** [comment]
Read https://en.wikipedia.org/wiki/Cross-validation_(statistics) under k-fold cross-validation.

**Comment by KervinBill** [comment]
@dutran do you mean train pre_trained models and extract features on UCF101's split1, split2, split3 respectively ? My database has 2000 videos. If I want to use 3-fold cross validation , what should I do ?
